tests/test_messages_documented.py

```diff
@@ -14,8 +14,14 @@
 README = CHANGELOG.parent / "README.md"
 
 # 107 & 108 are removed (but still mentioned in changelog & readme)
-# TRIOxxx_* are fake codes to get different error messages for the same code
-IGNORED_CODES_REGEX = r"TRIO107|TRIO108|TRIO\d\d\d_.*"
+# ASYNCxxx_* are fake codes to get different error messages for the same code
+IGNORED_CODES_REGEX = r"(TRIO|ASYNC)107|(TRIO|ASYNC)108|ASYNC\d\d\d_.*"
```

Comment:
```suggestion
IGNORED_CODES_REGEX = r"(TRIO|ASYNC)10[78]|ASYNC\d\d\d_.+"
```
I think this is net easier to read now